src-ghc/Pact/Types/Crypto.hs

```diff
@@ -116,21 +115,19 @@ class (ConvertBS (PublicKey a),  Eq (PublicKey a),
 
 
 
-
 --------- HASHING ---------
 
-hashTx :: (H.HashAlgorithm a) => ByteString -> a -> Hash
-hashTx b algo = (Hash . B.convert . H.hashWith algo) b
 
-verifyHashTx :: (H.HashAlgorithm a) => Hash -> ByteString -> a -> Either String Hash
-verifyHashTx h b algo = if hashTx b algo == h
-  then Right h
+verifyHashTx :: PactHash -> ByteString -> Either String Hash
```

Comment:
Why not delete and use `verifyHash` instead since `hashTx` has already been deleted? Or have `verifyHashTx` and `hashTx` shadow Pact.Types.Hash's equivalent functions?

Reply:
I'll remove them